pandas/io/gbq.py

```diff
@@ -125,6 +126,13 @@ def read_gbq(
         ``fastavro`` packages.
 
         .. versionadded:: 0.25.0
+    max_results : int, optional
+        If set, limit the maximum number of rows to fetch from the query
+        results.
```

Comment:
I just copied the docstring from pandas-gbq here, but given the current behaviour it might be an idea to add a line like `If set to 0, ``read_gbq`` will return ``None``.`